apps/blockchain/lib/blockchain/contract.ex

```diff
@@ -100,7 +101,7 @@ defmodule Blockchain.Contract do
       )
 
     {state_after_init, remaining_gas, accrued_sub_state, output} =
-      EVM.VM.run(available_gas, exec_env)
+      VM.run(available_gas, exec_env)
```

Comment:
style note. I think credo should fail with `pipe operator should start with raw value`

Reply:
Well, it doesn't work here because `VM.run` is not a "accepts the first argument as a special argument"-type function, so it would be weird to pipe in the raw value. We could change it to not use a pipe at all, but then we're going to end up with `nested(function(calls(v1, v2),v3))`, which I would argue is more difficult.